core/src/main/java/com/scalar/db/common/ActiveTransactionManagedDistributedTransactionManager.java

```diff
@@ -32,6 +34,17 @@ public abstract class ActiveTransactionManagedDistributedTransactionManager
 
   private final ActiveExpiringMap<String, ActiveTransaction> activeTransactions;
 
+  private final AtomicReference<BiConsumer<String, DistributedTransaction>>
+      transactionExpirationHandler =
+          new AtomicReference<>(
+              (id, t) -> {
+                try {
+                  t.rollback();
```

Comment:
[minor] I think this `rollback` is always necessary and any custom expiration handler must call it. But users of `setTransactionExpirationHandler()` can pass a custom expiration handler that doesn't call `rollback` by mistake. Another option would be to add a callback like this?
```java
private final AtomicReference<BiConsumer<String, DistributedTransaction>> transactionExpirationCallback = new AtomicReference<>((id, t) -> {});
private final BiConsumer<String, DistributedTransaction>
      transactionExpirationHandler =
              (id, t) -> {
                try {
                   transactionExpirationCallback.get().accept(id, t);
                }
                catch (Throwable t) {
                  // Warning
                }
                try {
                  t.rollback();
                } catch (Exception e) {
                  logger.warn("Rollback failed. transaction ID: {}", id, e);
                }
              };
```

Reply:
Good point. I agree that `rollback` is always necessary, which is why the default handler executes `rollback`. This feature is advanced and is primarily intended for internal use. Additionally, I don't plan to document it. Therefore, if a user sets a custom handler that doesn't rollback the expired transaction, I believe it falls under the user's responsibility. Thanks.